### DISCUSSION TITLE
Cooking Performance

### DISCUSSION THREAD

**Discussion by ayoubbelatrous** [jrouwe/JoltPhysics#570] [General]
I've been struggling to find a solution for generating collision shapes for quad tree planets in PhysX with a reasonable speed. I'm curious if Jolt Physics offers any improvements in that regard?
![image](https://github.com/jrouwe/JoltPhysics/assets/77838146/71a77da3-8f1e-409f-ba6b-86057597f810)

**Comment by jrouwe** [comment]
To be honest, I have never compared PhysX mesh creation speed with Jolt mesh creation speed, so I don't know if it is faster.

You could create a custom shape if you have your own acceleration structure already and that should reduce the build time to zero, but will obviously cost quite a bit of programming time. See [Create Custom Shapes on this page](https://jrouwe.github.io/JoltPhysics/).

**Reply by ayoubbelatrous** [reply]
thanks for the reply. I will definitely take a look at it.
